src/main/java/step2/lotto/game/Game.java

```diff
@@ -0,0 +1,44 @@
+package step2.lotto.game;
+
+import java.util.ArrayList;
+import java.util.List;
+import step2.lotto.domain.Generator;
+import step2.lotto.domain.Lotto;
+import step2.lotto.domain.Lottos;
+import step2.lotto.domain.Statistic;
+
+public class Game {
+
+    private final int LOTTO_PRICE = 1000;
+    private int gameCount;
+    private final Generator generator;
+    private Lottos lottos;
+
+    public Game() {
+        this.generator = new Generator();
+    }
+
+    public Lottos createLottos(int paidMoney) {
+        calculateLottoCount(paidMoney);
+        RollingLotto();
+        return lottos;
+    }
+
+    public void calculateLottoCount(int paidMoney) {
```

Comment:
```suggestion
    private void calculateLottoCount(int paidMoney) {
```
외부에서 사용하지 않는다면 private으로 노출을 최소화시키는 게 좋겠네요